app/src/main/java/com/topjohnwu/magisk/core/tasks/MagiskInstaller.kt

```diff
@@ -255,7 +255,7 @@ abstract class MagiskInstallImpl protected constructor(
                 val alpha = "abcdefghijklmnopqrstuvwxyz"
                 val alphaNum = "$alpha${alpha.toUpperCase(Locale.ROOT)}0123456789"
                 val random = SecureRandom()
-                val filename = StringBuilder("magisk_patched_").run {
+                val filename = StringBuilder("magisk_patched-${BuildConfig.VERSION_CODE}_").run {
```

Comment:
`magisk_patched-`  → `magisk_patched_`

Reply:
I've looked at it both ways and I think - looks better there:

magisk_patched_22005_31P8J.img
vs.
magisk_patched-22005_31P8J.img

Plus it matches the APK naming in a way: Magisk-f152b4c2(22005).apk